syntax = "proto2";
package oneflow;

message MsgEvent {
  required int64 src_actor_id = 1;
  required int64 dst_actor_id = 2;
  required double send_time = 3;
  required int64 piece_id = 4;
  required int64 act_id = 5;
  required int64 model_version_id = 6;
  required int64 regst_desc_id = 7;
  required int64 producer_actor_id = 8;
}
